### ISSUE TITLE
cast (feature request)

### ISSUE THREAD

**Issue by RekGRpth** [web2py/pydal#424]
``` python
db.define_table('table',
    Field('int_field', type='int'),
    Field('json_field', type='json'),
)
...
db.table.int_field.cast('text').startswith('1')
db.table.json_field.cast('text').contains('str')
```

**Comment by Marlysson** [comment]
This would be a query made in runtime isn't ?

Why do you not make query and make converting in the application.. you change to need format and using methods specifics of that format

**Comment by RekGRpth** [comment]
because it would be a database-side query

**Comment by Marlysson** [comment]
@gi0baro 
This was analysed to put in source code?

**Comment by gi0baro** [comment]
@Marlysson nope, there's no assignee to this.

**Comment by Marlysson** [comment]
@gi0baro But this already was discussed if will be accepted or not ?  Or something this ..

**Comment by gi0baro** [comment]
@Marlysson will be accepted as soon as someone take charge on this an write the code :)

**Comment by Marlysson** [comment]
@gi0baro 
Yep :smile: , seems the good feature to dal ..

**Comment by mdipierro** [comment]
+1

**Comment by timnyborg** [comment]
Implemented in #501
